docs/index.md

```diff
@@ -11,7 +11,20 @@ myst:
 
 # Plone 6 Documentation
 
-This website is the HTML documentation of Plone, an open source, enterprise-level content management system.
+This is the community-maintained documentation for the Plone content management system.
+
+This documentation is for:
+
+- Content editors who write, update, and order content on a site
```

Comment:
"order" is a strange verb here. Perhaps "design" or "lay out" instead?

```suggestion
- Content editors who write, update, and order content on a site
```

Reply:
You are good in finding appropriate wording. What's your first choice?

Reply:
Maybe "write, update, and review"

Reply:
@ksuess can you give me more context for your idea of using "order"? What words would you use in your native language? In English "order" can mean "to give a command" or "put into an arrangement".

@davisagli brings up the concept of workflows, such as "review" and "publish", but is that always the job of an Editor? For example, I can edit content on plone.org, but I don't have permission to publish it unless given explicit permission to do so. Maybe that's getting into the weeds and nitpicky. `¯\_(ツ)_/¯`

Reply:
Oh my, this is getting a big thing.

"order":  I suggest to skip this. I took the text from docs.plone.org. I think the arrangement of content is not a primary task. And this intro should be as short as possible. It's even now longish.

Why not take Davids "write, update, and review"? +1 from me.
Yes, an editor should not be reviewer. But often he is, on smaller sites.

Reply:
Heh, the curse of copy-pasta! We know that Plone 5 docs seldom went through quality control over the years, and until now, I never noticed that weird line.

Let's drop "order", remove repetition of "content", and move on. This is good enough for me.

`Editors who write and update content on a site`

Reply:
It would also be fine to just say "...who write and edit content on a site." The end-user docs can go into more detail. :)

I don't feel strongly about the wording, any of the above options would work and overall this intro is a nice improvement.